### ISSUE TITLE
Recent Travis-CI Build failures

### ISSUE THREAD

**Issue by chron0** [apollo-ng/dspace-app-apollong#86]
@elf-pavlik: Could you please investigate why this is happening?

```
npm ERR! Error: version not found: 1.1.26 : npm/1.1.26
npm ERR!     at RegClient.<anonymous> (/home/travis/.nvm/v0.8.22/lib/node_modules/npm/node_modules/npm-registry-client/lib/request.js:268:14)
npm ERR!     at Request.init.self.callback (/home/travis/.nvm/v0.8.22/lib/node_modules/npm/node_modules/request/main.js:120:22)
npm ERR!     at Request.EventEmitter.emit (events.js:99:17)
npm ERR!     at Request.<anonymous> (/home/travis/.nvm/v0.8.22/lib/node_modules/npm/node_modules/request/main.js:648:16)
npm ERR!     at Request.EventEmitter.emit (events.js:126:20)
npm ERR!     at IncomingMessage.Request.start.self.req.self.httpModule.request.buffer (/home/travis/.nvm/v0.8.22/lib/node_modules/npm/node_modules/request/main.js:610:14)
npm ERR!     at IncomingMessage.EventEmitter.emit (events.js:126:20)
npm ERR!     at IncomingMessage._emitEnd (http.js:366:10)
npm ERR!     at HTTPParser.parserOnMessageComplete [as onMessageComplete] (http.js:149:23)
npm ERR!     at CleartextStream.socketOnData (http.js:1485:20)
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>
```

**Comment by elf-pavlik** [comment]
i updated ender in 7a56c55 and now it doesn't error
@nilclass could you please take a look why tests fail? it complains about undefined plugin ...

**Comment by elf-pavlik** [comment]
fixed in 12c4f3868bc3
